src/environments/environment.ts

```diff
@@ -5,10 +5,10 @@
 
 export const environment = {
   production: false,
-  apiURL: 'http://localhost:3000',
+  apiURL: 'http://clark-gateway:3000',
```

Comment:
Are these temp values? How would this affect local dev?

Reply:
This is going to force it to be run in a docker network. Can we get around this through any build commands with the Angular CLI?

If not, maybe we need to add some proxy setup to the image.